### PR TITLE
[AD-250] Add POC jmeter test plan

### PR DESCRIPTION
**Pull request by andiemontoyeah**
### Summary

- Added JMeter directory with test plan and test data.
### Description

- Added .jmx file that executes test queries from a csv 
- Added csv with test queries and expected results 
- By changing user variables, can execute queries using DocumentDb JDBC driver or Mongo BI Connector + MySQL JDBC driver. 
### Related Issue

https://bitquill.atlassian.net/browse/AD-250 

### Additional Reviewers
@birschick-bq
@mitchell-elholm
@alexey-temnikov 
@andiem-bq

### CODE REVIEW

File: JMeter/JMeter.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# JMeter Tests
+
+## Set-Up
+
+### Install & Set Up JMeter   
+
+- Download [JMeter](http://jmeter.apache.org/download_jmeter.cgi).
```
Comment (alexey-temnikov) [inline JMeter/JMeter.md]: Add version what we are supporting.

File: JMeter/JMeter.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+- Add the [MySQL JDBC auth plugin](https://docs.mongodb.com/bi-connector/current/reference/auth-plugin-jdbc/)
```
Comment (alexey-temnikov) [inline JMeter/JMeter.md]: Should include only how to run it with DocumentDB in this repository.

File: JMeter/JMeter.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+- Add a [MySQL driver](https://dev.mysql.com/downloads/connector/j/) 
+  `.jar` to the JMeter `/lib` folder. 
+
+- Add the latest DocumentDbDriver `.jar` to the JMeter `/lib` folder.
+
+### Set Up a Test Cluster
+
+If targeting a cluster other than 
+`performance-cluster.cluster-c9voe85gof13.ca-central-1.docdb.amazonaws.com`,
```
Comment (alexey-temnikov) [inline JMeter/JMeter.md]: Doc should not contain specific endpoints.

File: JMeter/JMeter.md (hunk 2 above)
Comment (andiemontoyeah) [reply JMeter/JMeter.md]: Removed references to Mongo BI Connector specifically. Rephrased to say that the tests can with some setup be run with other JDBC drivers. :)

File: JMeter/DocumentDb_Test_Plan.jmx [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="USERNAME" elementType="Argument">
+            <stringProp name="Argument.name">USERNAME</stringProp>
+            <stringProp name="Argument.value">documentdb</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="PASSWORD" elementType="Argument">
+            <stringProp name="Argument.name">PASSWORD</stringProp>
+            <stringProp name="Argument.value">bqdocumentdblab</stringProp>
```
Comment (birschick-bq) [inline JMeter/DocumentDb_Test_Plan.jmx]: Not OK with clear text password! Can we get this and other properties via environment variables? Possibly using `${__groovy(System.getenv('username'),)}` - [pass-credentials-to-jmeter-command-line](https://stackoverflow.com/questions/65889891/pass-credentials-to-jmeter-command-line)

**Review by mitchell-elholm — COMMENTED** [review]
LGTM apart from the password issue Bruce mentioned.

File: JMeter/DocumentDb_Test_Plan.jmx (hunk 4 above)
Comment (andiemontoyeah) [reply JMeter/DocumentDb_Test_Plan.jmx]: good catch!! Changed to use _P() function to pass password from command line.

File: JMeter/JMeter.md [hunk 5]
```diff
@@ -18,16 +18,30 @@ A potential enhancement could be to automate this.
...
+   Change user variables `CONNECTION_STRING`, `USERNAME`, and `PASSWORD` depending on SSH tunnel setup and target cluster.
+
+1. Run the test plan. 
+
+#### Command-line 
+
+1. Start an SSH tunnel for target cluster. 
+
+1. Run the `DocumentDb_Test_Plan.jmx` file. 
+   Pass connection string (`JconnectionString`), username(`Jusername`), and password(`Jpassword`) as parameters. Connection string and username
```
Comment (birschick-bq) [inline JMeter/JMeter.md]: This isn't very clear. Could you add `-` to each of the options? I.e., `-JconnectionString`, etc.

**Review by birschick-bq — APPROVED** [review]
LGTM

**Review by mitchell-elholm — APPROVED** [review]
LGTM

**Pull request merged**